### ISSUE TITLE
Mathjax support

### ISSUE THREAD

**Issue by HaleTom** [plasticboy/vim-markdown#373]
I write a lot of formulae inside markdown.

Does this plugin syntax-highlight the LaTeX inside `$ ... $`  and `$$ ... $$`? Eg, stackoverflow highlights braces (`{`) and `\directives`.

Or does it define a syntax region for such sections which could be used to let another plugin (eg one specialising in latex syntax) do the syntax highlighting?

**Comment by gmarmstrong** [comment]
Yes, with `let g:vim_markdown_math = 1`. It also supports concealing formulas (displaying superscript, Greek letters, etc.) with `let conceallevel=2`.

For example, `$3_x + z^n \cdot \alpha$` displays as `3ₓ + zⁿ · α`.
